understanding/section-headings.html

```diff
@@ -58,141 +56,117 @@
-                  <dd>Users understand how content is organized in sections.</dd>
+                  <dt>目標</dt>
+                  <dd>コンテンツがセクションによってどのように構成されているかを、利用者が理解している。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Where content is organized in sections, provide section headings.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>コンテンツがセクションによって構成されている場合は、セクション見出しを付ける。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>People can orient themselves, especially those with cognitive or visual disabilities.</dd> 
+                  <dt>なぜそれが重要か</dt>
+                  <dd>特に認知障害又は視覚障害のある人々が、ページ上の自分の位置を確認することができる。</dd> 
                   
                </dl>
             </section>
             <section id="intent">
-               <h2>Intent</h2>
-               <p>The intent of this Success Criterion is to provide headings for sections of a Web
-                  page, when the page is organized into sections. For instance, long documents are often
-                  divided into a variety of chapters, chapters have subtopics, etc. When such sections
-                  exist,
-                  they need to have headings that introduce them. This clearly indicates the organization
-                  of the content, facilitates navigation within the content, and provides mental "handles"
-                  that aid in comprehension of the content. Other page elements may complement headings
-                  to improve presentation (e.g., horizontal rules and boxes), but visual presentation
-                  is not sufficient to identify document sections.
+               <h2>意図</h2>
+               <p>この達成基準の意図は、ページがセクションで構成されている場合に、ウェブページのセクションに見出しを提供することである。例えば、長い文書は多くの場合、様々な章に分けられ、章にはサブトピックなどがある。そのようなセクションが存在する場合、それらのセクションを紹介する見出しが必要である。これは、コンテンツの構造を明確に示し、コンテンツ内のナビゲーションを手助けし、そしてコンテンツの理解を助けるメンタルな「取っ掛かり」を提供する。提示を改善するために、ページのその他の要素 (例: 横罫線、囲み罫線) が見出しを補完することもあるが、視覚的提示は文章のセクションを特定するには十分ではない。
                   
                </p>
-               <p>This provision is included at Level AAA because it cannot be applied to all types
-                  of content and it may not always be possible to insert headings. For example, when
-                  posting a pre-existing document to the Web, headings that an author did not include
-                  in the original document cannot be inserted. Or, a long letter would often cover different
-                  topics, but putting headings into a letter would be very strange. However, if a document
-                  can be broken up into sections with headings, it facilitates both understanding and
-                  navigation.
+               <p>この達成基準は、すべての種類のコンテンツに適用できず、見出しを挿入することが常に可能であるとは限らないため、レベル AAA に含まれている。例えば、既存の文書をウェブで公開する場合、元の文書に含まれていない見出しは挿入できない。又は、長い手紙には様々なトピックを扱うことが多いが、手紙に見出しを付けるととてもおかしくなってしまう。しかし、文書が見出しの付いたセクションに分割できる場合、文書は理解とナビゲーションの両方が容易になる。
```

Comment:
「すべての種類のコンテンツに適用できず」は「どんなコンテンツにも適用できない」とも読めるので少し調整したいです。

putting headings into a letter would be very strange は putting headings という行為が strange であると言っているように思います。

こんな感じでどうでしょうか。

```suggestion
               <p>この達成基準は、すべての種類のコンテンツに適用できるわけではなく、見出しを挿入することが常に可能であるとも限らないため、レベル AAA に含まれている。例えば、既存の文書をウェブで公開する場合、元の文書に含まれていない見出しは挿入できない。又は、長い手紙には様々なトピックを扱うことが多いが、手紙に見出しを付けるのはとても奇妙なことだろう。しかし、もし文書が見出しの付いたセクションに分割できるならば、理解とナビゲーションの両方が容易になる。
```